cern_access/indico_cern_access/util.py

```diff
@@ -358,5 +363,11 @@ def sanitize_personal_data():
     db.session.flush()
 
 
+def sanitize_license_plate(number):
+    """Sanitize a license plate number to [A-Z0-9]+, no dashes/spaces."""
+    number = re.sub(r'[ -]', '', number.upper())
+    return number if re.match(r'[A-Z0-9]+', number) else None
```

Comment:
doesn't this need `$` at the end? (and maybe `^` at the start to be consistent even though `.match()` only searches at the beginning anyway)